nodejs/week3/src/backend/api/meals.js

```diff
@@ -0,0 +1,148 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+const knex = require("../database");
+
+//Get all meals
+router.get("/", async (req, res) => {
+  try {
+    const mealslist = await knex("meals").select("*");
+    res.send(mealslist);
+  } catch (error) {
+    throw error;
+  }
+});
```

Comment:
you only need to have a single request handler for the '/meals/' route (you already have one handler below so you can remove this one)

Reply:
thanks , i will do so